node/src/accountant/mod.rs

```diff
@@ -2279,16 +2340,29 @@ mod tests {
 
         send_start_message!(subject_subs);
 
+        let time_before = SystemTime::now();
         system.run();
+        let time_after = SystemTime::now();
         //the second attempt is the one where the queue is empty and System::current.stop() ends the cycle
-        let begin_scan_params = begin_scan_params_arc.lock().unwrap();
         let notify_later_payables_params = notify_later_payables_params_arc.lock().unwrap();
         TestLogHandler::new().exists_log_containing(&format!(
             "DEBUG: {test_name}: There was nothing to process during Payables scan."
         ));
+        let mut begin_scan_params = begin_scan_params_arc.lock().unwrap();
+        let first_begin_scan_params = begin_scan_params.remove(0);
+        let (wallet_1, timestamp_1, response_skeleton_1, logger_1) = first_begin_scan_params;
```

Comment:
Bert thinks these wallet names could be better.

Maybe `first_attempt_wallet` and `second_attempt_wallet`. They're the same wallet, just two different calls with that wallet.